examples/redux/ngrx-actions/app/index.ts

```diff
@@ -1,11 +1,23 @@
-import { bootstrap } from '@angular/platform-browser-dynamic';
-import { provide } from '@angular/core';
-import { provideStore } from '@ngrx/store';
+import { NgModule } from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+import { StoreModule } from '@ngrx/store';
 
 import { SimpleNgrx } from './containers/app-container';
+import { Counter } from './components/counter-component';
 import { counterReducer } from './reducers/counter-reducer';
 
-bootstrap(SimpleNgrx, [
-  provideStore({ counter: counterReducer })
-]);
-
+@NgModule({
```

Comment:
Ah, was trying to keep an eye on `Ngrx` vs `Redux` - do the NGRX the NGRX way then :)